secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala

```diff
@@ -77,8 +75,6 @@ class FeedbackContext[CT,CV](user: String,
   private def replayComputation(reason: String, resync: Boolean): ReplayLater = {
     ReplayLater(reason, { feedbackCookie =>
```

Comment:
This is turned into an exception over here

https://github.com/socrata-platform/data-coordinator/blob/main/secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackSecondary.scala#L167

and that exception is caught and turned into a limited-retry here:

https://github.com/socrata-platform/data-coordinator/blob/main/secondarylib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala#L112

Reply:
Fun fact, if you use a sha instead of `main`, you get this:

https://github.com/socrata-platform/data-coordinator/blob/cff0665bf309bf55f6c21c7c7124baf2796dd8fd/secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackSecondary.scala#L167

https://github.com/socrata-platform/data-coordinator/blob/cff0665bf309bf55f6c21c7c7124baf2796dd8fd/secondarylib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala#L112